src/device.js

```diff
@@ -16,14 +16,29 @@ export class Device {
 }
 
 export class DeviceManager {
+    static DEVICE_REFRESH_INTERVAL_MIN = 30;
```

Comment:
static or rather const?

Reply:
In ES6 const cannot be used in this context. So either we keep it like that (as we do in the rest of the codebase) or we move it to global scope.